spec/factories/chunks.rb

```diff
@@ -4,5 +4,9 @@
     vector_id { SecureRandom.uuid }
     content { Faker::Lorem.sentence(word_count: 10) }
     embedding_content { Faker::Lorem.sentence(word_count: 16) }
+
+    factory :chunk_from_web do
```

Comment:
Does this still work if you move it out of the `factory :chunk do` block?

Reply:
Yup. Looks like I just add `parent: :chunk`.
I'll fix that, and the tests soon.